ddl/ddl.go

```diff
@@ -150,14 +150,15 @@ func (e *Event) String() string {
 	return ret
 }
 
+// ddl is data definition language which defines basic operation such as create, alter and select.
```

Comment:
select is not DDL, but DML.

ddl represents the statements which are used to define the database structure or schema.